src/elf.rs

```diff
@@ -382,10 +388,11 @@ struct Elf<'a> {
     sizeof_strtab: Offset,
     strings: DefaultStringInterner,
     sizeof_bits: Offset,
-    nsections: u16,
+    nsections: u32,
```

Comment:
This should be a u64 now (but I don’t think this is causing the breaking issue)

Reply:
the symbol indices table uses u32s, so we're still limited to 32-bit

Reply:
I guess `2^32` sections should be enough for anyone ;)